client/src/modules/shopping-list/index.jsx

```diff
@@ -87,33 +111,61 @@ class ShoppingList extends Component {
       list
     } = this.props;
     const listItems = list && list.products ? list.products : [];
-    const archiveList = listItems.filter(item => item.isOrdered);
+    const archivedList = listItems.filter(item => item.isOrdered);
     const shoppingList = listItems.filter(item => !item.isOrdered);
     const description = list && list.description ? list.description : null;
+    const isArchived = list && list.isArchived ? list.isArchived : null;
     const name = list && list.name ? list.name : null;
 
     return (
       <Fragment>
         <Toolbar>
-          <ToolbarItem mainIcon={<EditIcon />} onClick={this.showUpdateForm} />
-          <ToolbarItem mainIcon={<RemoveIcon />} onClick={this.showDialogBox} />
+          {!isArchived && (
+            <Fragment>
```

Comment:
do we really need fragment here as react only warns when he found two child elements at the top level of render method??

Reply:
Yes, we need it, without fragment we get error...